webapp/Makefile

```diff
@@ -1,8 +1,14 @@
 ifndef TOPDIR
 TOPDIR=..
 endif
+
+REC_TARGETS = domserver
+
 include $(TOPDIR)/Makefile.global
 
+# Subdirectories to recurse into for REC_TARGETS
+SUBDIRS = app
```

Comment:
Given that we now have a `Makefile` under `webapp/app` I think it makes sense to move all things rules that target files under there from `webapp/Makefile` into `webapp/app/Makefile`.